python/cuml/dask/datasets/classification.py

```diff
@@ -0,0 +1,232 @@
+        hypercube.
+    n_redundant : int, optional (default=2)
+        The number of redundant features. These features are generated as
+        random linear combinations of the informative features.
+    n_repeated : int, optional (default=0)
+        The number of duplicated features, drawn randomly from the informative
+        and the redundant features.
+    n_classes : int, optional (default=2)
+        The number of classes (or labels) of the classification problem.
+    n_clusters_per_class : int, optional (default=2)
+        The number of clusters per class.
+    weights : array-like of shape (n_classes,) or (n_classes - 1,),\
+              (default=None)
+        The proportions of samples assigned to each class. If None, then
+        classes are balanced. Note that if ``len(weights) == n_classes - 1``,
+        then the last class weight is automatically inferred.
+        More than ``n_samples`` samples may be returned if the sum of
+        ``weights`` exceeds 1.
+    flip_y : float, optional (default=0.01)
+        The fraction of samples whose class is assigned randomly. Larger
+        values introduce noise in the labels and make the classification
+        task harder.
+    class_sep : float, optional (default=1.0)
+        The factor multiplying the hypercube size.  Larger values spread
+        out the clusters/classes and make the classification task easier.
+    hypercube : boolean, optional (default=True)
+        If True, the clusters are put on the vertices of a hypercube. If
+        False, the clusters are put on the vertices of a random polytope.
+    shift : float, array of shape [n_features] or None, optional (default=0.0)
+        Shift features by the specified value. If None, then features
+        are shifted by a random value drawn in [-class_sep, class_sep].
+    scale : float, array of shape [n_features] or None, optional (default=1.0)
+        Multiply features by the specified value. If None, then features
+        are scaled by a random value drawn in [1, 100]. Note that scaling
+        happens after shifting.
+    shuffle : boolean, optional (default=True)
+        Shuffle the samples and the features.
+    random_state : int, RandomState instance or None (default)
+        Determines random number generation for dataset creation. Pass an int
+        for reproducible output across multiple function calls.
+        See :term:`Glossary <random_state>`.
+    order: str, optional (default='F')
+        The order of the generated samples
+    dtype : str, optional (default='float32')
+        Dtype of the generated samples
+    n_parts : int (default = None)
+        number of partitions to generate (this can be greater
+        than the number of workers)
+    Returns
+    -------
+    X : device array of shape [n_samples, n_features]
+        The generated samples.
+    y : device array of shape [n_samples]
+        The integer labels for class membership of each sample.
+    Notes
```

Comment:
I made a more detailed comment about this below- but we probably want to expend a sentences explaining our general approach- specifically how we are distributing the algorithm w/ Dask.